libvast/include/vast/system/evaluator.hpp

```diff
@@ -76,6 +83,7 @@ struct evaluator_state {
 /// @pre `!eval.empty()`
 evaluator_actor::behavior_type
 evaluator(evaluator_actor::stateful_pointer<evaluator_state> self,
-          expression expr, std::vector<evaluation_triple> eval);
+          expression expr, std::vector<evaluation_triple> eval,
+          ids all_possible_ids_for_partition);
```

Comment:
Maybe this should be `optional<ids>` since you sometimes also send an empty `ids{}`

Reply:
I have changed variable name so it no longer "lies"